tyk-stack/README.md

```diff
@@ -188,6 +87,17 @@ helm show values tyk-helm/tyk-stack > values.yaml
 You can update any value in your local `values.yaml` file and use `-f [filename]` flag to override default values during installation. 
 Alternatively, you can use `--set` flag to set it in Tyk installation. See [Using Helm](https://helm.sh/docs/intro/using_helm/) for examples.
 
+To configure Tyk components, users can utilize both config files and [environment variables](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/). Notably, environment variables take precedence over config files. To maintain simplicity and consistency, the Tyk Helm Charts deploy components with an empty config file while setting container environment variables based on user-defined [values](https://helm.sh/docs/chart_best_practices/values/). This approach ensures seamless integration with Kubernetes practices, allowing for efficient management of configurations. For a comprehensive overview of available configurations, please refer to the [configuration documentation](https://tyk.io/docs/tyk-environment-variables/). Additionally, should any environment variables not be set by the Helm Chart, users can easily add them under the `extraEnvs` section within the charts for further customization. Values set under `extraEnvs` would take precedence over all configurations.
+
+```yaml
+tyk-gateway:
```

Comment:
What do you think about adding a small statement to emphasize that the following example shows how to add environment variables to only Tyk Gateway (not Dashboard)? if users would like to add environment variables to other components, they need to update the corresponding component's extraEnvs section

Reply:
thank you @buraksekili . Added a note above the example to explain that.

Reply:
thanks!